### PR TITLE
Added symbolserver support to cocoa symbolication

### PR DESCRIPTION
**Pull request by mitsuhiko**
This pull request adds support for the new symbol server.  So far it's opt-in
which is why the paths are currenlty quite ugly in the code.

This PR also removes the internal cache handling for system frames since this
is now managed by the outer cache anyways we added earlier.

### CODE REVIEW

**Comment by mitsuhiko** [comment]
@mattrobenolt so i think we can merge this and keep it disabled until we have the server deployed. Then probably start a worker with the feature on and see how it does, then switch over.

File: src/sentry/lang/native/systemsymbols.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    symbol_query = {
+        'sdk_id': sdk_info_to_sdk_id(sdk_info),
+        'cpu_name': cpu_name,
+        'symbols': symbols,
+    }
+
+    attempts = 0
+    while 1:
+        try:
+            rv = safe_urlopen('%s/lookup' % settings.SYMBOL_SERVER_URL.rstrip('/'),
```
Comment (mattrobenolt) [inline src/sentry/lang/native/systemsymbols.py]: We're not going to be able to use `safe_urlopen`. I'd suggest using urllib3 directly like I do for the react backend. Or at least, just using requests directly.

`safe_urlopen` is going to block access to this being an internal service.

**Comment by mitsuhiko** [comment]
Indeed i will fix that

**Comment by mitsuhiko** [comment]
@mattrobenolt this should be fixed now.

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**